### DISCUSSION TITLE
what's the maximum number added to offline logstorage dlt file? (e.g. DLT_LOGFILE.XXX.dlt)

### DISCUSSION THREAD

**Discussion by ynakamura4** [COVESA/dlt-daemon#597] [Q&A] [answered]
Hello all, 

I would like to know the maxmum number which is added to offline stroage dlt file. 
When we define dlt_logstorage.conf like below dlt file like DLT_LOGFILE.1.dlt is created. 

[FILTER]
LogAppName=.*
ContextName=TEST
LogLevel=DLT_LOG_INFO
File=DLT_LOGFILE
FileSize=524288
NOFiles=5

the number of log file  is incremented by dlt-daemon. 

Can you tell me the maximum number of XXX (DLT_LOGFILE.XXX.dlt) ?

**Comment by minminlittleshrimp** [comment, marked as answer]
Hello @ynakamura4 

Thank you for dropping nice question here. The maximum counter value would be by default 999, and you can modify this value in dlt.conf: OfflineLogstorageMaxCounter

Hope this answer the question.
Regards
Minh

**Reply by ynakamura4** [reply]
Hello Minh-san!

thank you very much for the quick reply. 

ok. does it return to "0" after "999"?

Best Regard,

**Reply by minminlittleshrimp** [reply]
Yes @ynakamura4 
That's the way the counter reset.

**Comment by ynakamura4** [comment]
Thank you very much for the clarification.
